types/elsys/ers-co2-lite/uplink.js

```diff
@@ -476,7 +476,7 @@ function consume(event) {
     }
 
     if (deleteUnusedKeys(noise)) {
-      emit("sample", { data: noise, topic: "noise" });
+      emit("sample", { data: noise, topic: "sound" });
```

Comment:
this is a breaking change - should backwards compatibility be ensured?

Reply:
I will give out both for now but hide noise from now on. In the DTL 2.0 we probably will unify it.